lib/components/viewers/stop-viewer.js

```diff
@@ -156,11 +157,23 @@ class StopViewer extends Component {
     zoomToPlace(map, stopData)
   }
 
-  _renderHeader = () => {
-    const { enableFavoriteStops, hideBackButton, showUserSettings, stopData } =
-      this.props
+  _renderHeader = (agencyCount) => {
+    const {
+      enableFavoriteStops,
+      hideBackButton,
+      showUserSettings,
+      stopData,
+      transitOperators
+    } = this.props
+
+    // We can use the first route, as this operator will only be used if there is only one operator
+    const stationOperator = transitOperators.find(
+      (o) =>
+        o.agencyId === stopData?.agencyId || stopData?.routes?.[0]?.agencyId
```

Comment:
Surround the `||` statement with parentheses (try with this stop http://localhost:9966/#/stop/2:841 to see the difference).

Reply:
Very good catch thank you!